frontend/src/components/Sidebar/MapComponents/CustomControl.tsx

```diff
@@ -0,0 +1,134 @@
+// SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+// Third-party
+import * as am5map from '@amcharts/amcharts5/map';
+import {ZoomIn, ZoomOut, ZoomOutMap} from '@mui/icons-material';
+import {ButtonGroup, IconButton, Stack} from '@mui/material';
```

Comment:
Please import each Type/Class seperately. Please look at the documentation here: https://github.com/DLR-SC/ESID/tree/develop/frontend#imports

Reply:
Thanks for the info, fixed.
https://github.com/DLR-SC/ESID/pull/384/commits/9f9a658e46c5e7673927bd331d99dd6d6542bf53